src/ws/main.c

```diff
@@ -39,6 +39,10 @@
 #include <libssh/libssh.h>
 #include <libssh/callbacks.h>
 
+#if WITH_DEBUG
```

Comment:
I don't think we need this condition. Is there a problem with just including the header?